### PR TITLE
Added some output to aab and improved the false positives a bit

### PR DESCRIPTION
**Pull request by rlaemmert**

### CODE REVIEW

File: libr/core/blaze.c [hunk 1]
```diff
@@ -435,18 +446,21 @@ R_API bool core_anal_bbs(RCore *core, const char* input) {
...
+					}
 				}
+				fcnFree (current_function);
 			}
 
 			r_stack_free (stack);
-			fcnFree (current_function);
 		}
 	}
+	eprintf ("DONE\n");
```
Comment (radare) [inline libr/core/blaze.c]: uh leftover here

**Comment by radare** [comment]
those eprintfs are a bit noisy, wanna add them under aabv? (for verbose only mode?)

**Comment by radare** [comment]
@Maijin

**Comment by Maijin** [comment]
What about anal.verbose ?

**Comment by radare** [comment]
there was cfg.debug at the time, but nobody uses it at all, but that was the plan for it

**Pull request merged**